tests/baselines/reference/goToTypeDefinition_arrayType.baseline.jsonc

```diff
@@ -0,0 +1,4564 @@
+// === goToType ===
+// === /tests/cases/fourslash/goToTypeDefinition_arrayType.ts ===
+// type User = [|{| defId: 0 |}{ name: string }|];
+// declare const users: User[]
+// /*GOTO TYPE*/users
+
+// === lib.d.ts ===
```

Comment:
@sheetalkamat this baseline shows an unfortunate edge case. I thought we discussed whether less than a full file could be printed back, but I don’t remember the outcome. Is there anything we can do about this?

Reply:
Could the test include custom lib.d.ts

Reply:
Can we not just truncate lines +/- _N_ away from markers?